pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.facebook.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>100</version>
+        <version>101</version>
```

Comment:
is this change related?

Reply:
Yes I needed to upgrade git-commit-id-plugin because it errors out in GH actions because of the settings GH actions uses to clone the repository.